### PR TITLE
i18n(zh-cn): translate from-vuepress.mdx

### PR DESCRIPTION
**Pull request by woshierha**
#### What kind of changes does this PR include?

- Translated content

#### Description

- translate from-vuepress.mdx

### CODE REVIEW

**Comment by liruifengv** [comment]
Thank you for your contribution. the description of the PR needs to be modified a bit, select the category you are modifying this time, and the description of what you are modifying.

For example:

### What kind of changes does this PR include?
Translated content
### Description
translate from-vuepress.mdx

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-vuepress.mdx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+---
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+[VuePress](https://vuePress.vuejs.org) 是一个基于 Vue 构建的开源静态站点生成器.
+
+## VuePress 和 Astro 的主要相似之处
+
+VuePress 和 Astro 有一些共同的特点，这些相似之处将有助于你迁移项目：
+
+- VuePress 和 Astro 都是现代的 Javascript 静态网站生成器,具有类似的项目文件结构。它们都使用 [特殊的`src/pages/`文件夹来进行基于文件的路由](/en/core-concepts/astro-pages/)。因此，在创建和管理站点页面时，你应该会感到非常熟悉。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-vuepress.mdx:17]:
```suggestion
- VuePress 和 Astro 都是现代的 JavaScript 静态网站生成器，具有类似的项目文件结构。它们都使用 [特殊的`src/pages/`文件夹来进行基于文件的路由](/zh-cn/core-concepts/astro-pages/)。因此，在创建和管理站点页面时，你应该会感到非常熟悉。
```
The link should be pointed to zh-cn.

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-vuepress.mdx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+- Astro 和 VuePress 都是为 [以内容为中心的网站](/en/concepts/why-astro/#content-focused) 设计的，它们都拥有出色的内置 Markdown 文件支持。因此，在使用 Markdown 编写内容时，你会感到非常熟悉，并且你可以保留现有的内容。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-vuepress.mdx:19]:
```suggestion
- Astro 和 VuePress 都是为 [以内容为中心的网站](/zh-cn/concepts/why-astro/#以内容为中心) 设计的，它们都拥有出色的内置 Markdown 文件支持。因此，在使用 Markdown 编写内容时，你会感到非常熟悉，并且你可以保留现有的内容。
```

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-vuepress.mdx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+- Astro 提供了 [官方集成的 Vue 组件使用方式](/en/guides/integrations-guide/vue/) 并且支持 [安装NPM包](/en/guides/imports/#npm-packages)，其中包括一些适用于 Vue 的包。你可以编写 Vue UI 组件，并且可能可以保留你现有的部分或全部 Vue 组件和依赖项。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-vuepress.mdx:21]:
```suggestion
- Astro 提供了 [官方集成的 Vue 组件使用方式](/zh-cn/guides/integrations-guide/vue/) 并且支持 [安装NPM包](/zh-cn/guides/imports/#npm-包)，其中包括一些适用于 Vue 的包。你可以编写 Vue UI 组件，并且可能可以保留你现有的部分或全部 Vue 组件和依赖项。
```

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-vuepress.mdx [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+
+## VuePress 和 Astro 的主要区别
+
+当你将 VuePress 站点重构为 Astro 站点时，你会注意到一些重要的差异：
+
+- [MPA 和 SPA](/en/concepts/mpa-vs-spa/): VuePress 是基于 Vue 的单页面应用(SPA)。而 Astro 站点是使用 [`.astro`组件](/en/core-concepts/astro-components/)构建的多页面应用(MPA)，同时也支持 [React, Preact, Vue.js, Svelte, SolidJS, AlpineJS, Lit](/en/core-concepts/framework-components/) 和原生 HTML 模板。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-vuepress.mdx:28]:
```suggestion
- [MPA 和 SPA](/en/concepts/mpa-vs-spa/): VuePress 是基于 Vue 的单页面应用（SPA）。而 Astro 站点是使用 [`.astro`组件](/zh-cn/core-concepts/astro-components/)构建的多页面应用（MPA），同时也支持 [React, Preact, Vue.js, Svelte, SolidJS, AlpineJS, Lit](/zh-cn/core-concepts/framework-components/) 和原生 HTML 模板。
```

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-vuepress.mdx [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+- [模板布局](/en/core-concepts/layouts/): VuePress 站点使用 Markdown (`.md`) 文件作为页面内容，同时使用HTML (`.html`) 模板作为布局。而 Astro 则是基于组件的，使用了 Astro 组件来构建页面的HTML模板、布局以及单个的UI元素。Astro 也可以通过[`.md` and `.mdx` 文件](/en/guides/markdown-content/) 来创建页面，通过 Astro 布局组件将 Markdown 内容包装在页面模板中。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-vuepress.mdx:30]:
```suggestion
- [模板布局](/zh-cn/core-concepts/layouts/): VuePress 站点使用 Markdown (`.md`) 文件作为页面内容，同时使用 HTML (`.html`) 模板作为布局。而 Astro 则是基于组件的，使用了 Astro 组件来构建页面的HTML模板、布局以及单个的 UI 元素。Astro 也可以通过[`.md` and `.mdx` 文件](/zh-cn/guides/markdown-content/) 来创建页面，通过 Astro 布局组件将 Markdown 内容封装在页面模板中。
```

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-vuepress.mdx [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+
+- VuePress 是为内容丰富的 Markdown 优先网站设计的，并且它还具有一些内置的、专为文档设计的网站功能，而这些功能需要你自己在 Astro 中构建。同时，Astro 也提供了一些专为文档编写的功能，通过 [官方文档主题](https://github.com/withastro/astro/tree/latest/examples/docs?on=github) 就能够实现。本网站就是使用该模板构建的！ 你还可以在我们的主题展示页面中找到更多内置各种功能的 [社区文档主题](https://astro.build/themes?search=&categories%5B%5D=docs)。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-vuepress.mdx:32]: [官方文档主题](https://github.com/withastro/astro/tree/latest/examples/docs?on=github) Link had be removed. Maybe we should update the content here. @Yan-Thomas

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-vuepress.mdx [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+
+## 从 VuePress 迁移到 Astro 的步骤
+
+要将 VuePress 文档网站转换为 Astro ，可以通过我们的官方 [Starlight 文档主题起始模板](https://starlight.astro.build)开始创建，或者在我们的 [主题展示页面](https://astro.build/themes?search=&categories%5B%5D=docs) 中探索更多社区文档主题。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-vuepress.mdx:36]:
```suggestion
要将 VuePress 文档网站转换为 Astro ，可以通过我们的官方 [Starlight 文档主题起始模板](https://starlight.astro.build/zh/)开始创建，或者在我们的 [主题展示页面](https://astro.build/themes?search=&categories%5B%5D=docs) 中探索更多社区文档主题。
```

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-vuepress.mdx [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+
+你可以在 `create astro` 命令中使用 `--template` 参数，并选择其中一个官方起始模板来开始一个新的 Astro 项目。另外，你也可以 [从 GitHub 上的任意现有 Astro 存储库开始一个新项目](/en/install/auto/#starter-templates)。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-vuepress.mdx:38]:
```suggestion
你可以在 `create astro` 命令中使用 `--template` 参数，并选择其中一个官方起始模板来开始一个新的 Astro 项目。另外，你也可以 [从 GitHub 上的任意现有 Astro 存储库开始一个新项目](/zh-cn/install/auto/#入门模板)。
```

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-vuepress.mdx [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+    ```
+    </Fragment>
+    <Fragment slot="yarn">
+    ```shell
+    yarn create astro --template starlight
+    ```
+    </Fragment>
+  </PackageManagerTabs>
+
+带着你现有的 Markdown 内容来 [创建 Markdown 页面](/en/guides/markdown-content/)。 你仍然可以利用 [基于文件的路由](/en/core-concepts/routing/)，只需将这些文档从 VuePress 的 `docs` 文件目录下 移动到 Astro 的 `src/pages/` 目录下，并且保证创建了与现有的 VuePress 项目对应的文件夹，就能够保留现有的URLs.
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-vuepress.mdx:58]:
```suggestion
带着你现有的 Markdown 内容来 [创建 Markdown 页面](/zh-cn/guides/markdown-content/)。 你仍然可以利用 [基于文件的路由](/zh-cn/core-concepts/routing/)，只需将这些文档从 VuePress 的 `docs` 文件目录下 移动到 Astro 的 `src/pages/` 目录下，并且保证创建了与现有的 VuePress 项目对应的文件夹，就能够保留现有的URLs. 
```

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-vuepress.mdx [hunk 10]
```diff
@@ -0,0 +1,66 @@
...
+    <Fragment slot="yarn">
+    ```shell
+    yarn create astro --template starlight
+    ```
+    </Fragment>
+  </PackageManagerTabs>
+
+带着你现有的 Markdown 内容来 [创建 Markdown 页面](/en/guides/markdown-content/)。 你仍然可以利用 [基于文件的路由](/en/core-concepts/routing/)，只需将这些文档从 VuePress 的 `docs` 文件目录下 移动到 Astro 的 `src/pages/` 目录下，并且保证创建了与现有的 VuePress 项目对应的文件夹，就能够保留现有的URLs. 
+
+VuePress 或你安装的其他任意主题可能已经为你处理了大部分站点的布局和元数据。你可能希望阅读关于 [在 Astro 中将 Markdown 页面包装为布局](/en/core-concepts/layouts/#markdownmdx-layouts) 以了解如何在 Astro 中自行管理模板，包括页面的 `<head>` 部分。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-vuepress.mdx:60]:
```suggestion
VuePress 或你安装的其他任意主题可能已经为你处理了大部分站点的布局和元数据。你可能希望阅读关于 [在 Astro 中将 Markdown 页面封装为布局](/zh-cn/core-concepts/layouts/#markdownmdx-布局) 以了解如何在 Astro 中自行管理模板，包括页面的 `<head>` 部分。
```

**Review by liruifengv — CHANGES_REQUESTED** [review]

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-vuepress.mdx (hunk 6 above)
Comment (Yan-Thomas) [reply src/content/docs/zh-cn/guides/migrate-to-astro/from-vuepress.mdx:32]: Thanks for pointing out!

**Review by liruifengv — APPROVED** [review]
LGTM

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-vuepress.mdx [hunk 11]
```diff
@@ -0,0 +1,66 @@
...
+- Astro 和 VuePress 都是为 [以内容为中心的网站](/zh-cn/concepts/why-astro/#以内容为中心) 设计的，它们都拥有出色的内置 Markdown 文件支持。因此，在使用 Markdown 编写内容时，你会感到非常熟悉，并且你可以保留现有的内容。
+
+- Astro 提供了 [官方集成的 Vue 组件使用方式](/zh-cn/guides/integrations-guide/vue/) 并且支持 [安装NPM包](/zh-cn/guides/imports/#npm-包)，其中包括一些适用于 Vue 的包。你可以编写 Vue UI 组件，并且可能可以保留你现有的部分或全部 Vue 组件和依赖项。
+
+
+## VuePress 和 Astro 的主要区别
+
+当你将 VuePress 站点重构为 Astro 站点时，你会注意到一些重要的差异：
+
+- [MPA 和 SPA](/en/concepts/mpa-vs-spa/): VuePress 是基于 Vue 的单页面应用（SPA）。而 Astro 站点是使用 [`.astro`组件](/zh-cn/core-concepts/astro-components/)构建的多页面应用（MPA），同时也支持 [React, Preact, Vue.js, Svelte, SolidJS, AlpineJS, Lit](/zh-cn/core-concepts/framework-components/) 和原生 HTML 模板。
```
Comment (Yan-Thomas) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-vuepress.mdx:28]:
```suggestion
- [MPA 和 SPA](/zh-cn/concepts/mpa-vs-spa/): VuePress 是基于 Vue 的单页面应用（SPA）。而 Astro 站点是使用 [`.astro`组件](/zh-cn/core-concepts/astro-components/)构建的多页面应用（MPA），同时也支持 [React, Preact, Vue.js, Svelte, SolidJS, AlpineJS, Lit](/zh-cn/core-concepts/framework-components/) 和原生 HTML 模板。
```

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Great work :purple_heart:

**Pull request merged**